src/window/mapped.rs

```diff
@@ -108,7 +111,7 @@ niri_render_elements! {
 
 static MAPPED_ID_COUNTER: IdCounter = IdCounter::new();
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Ord, PartialOrd)]
```

Comment:
No, these are definitely not intended to be Ord/PartialOrd. If you replace BinaryHeap with Vec like I mentioned above, this will not be necessary.